Update README.md (#58)
contao/managed-edition: README.md

 * Web server
 * PHP 7.3+ with GDlib and DOM

>  * MySQL 5.5.7+ (MySQL Strict Mode is recommended)

Reviewer: ```suggestion
 * MySQL 5.5.7+
```

We should keep this as clear as possible IMHO.

                    engine: MyISAM
```

## MySQL Strict Mode

It is recommended to run MySQL in "strict mode" to prevent corrupt or truncated
data and to guarantee data integrity. As of Contao 4.12, the install tool shows
a warning if the database server is not running in strict mode. To enable it,
add the following to your `my.cnf` file or make sure that the setting is adjusted
accordingly:

```
[mysqld]
…
sql_mode="STRICT_TRANS_TABLES"
…
```

If the setting cannot be enabled on your server, please configure the connection
options in your `config/config.yml` file:

```yml
doctrine:
    dbal:
        connections:
            default:
                options:

>                     1002: "SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))"

Reviewer 2: This option is `1002` only if PDO is used. For mysqli the option is named `3`, see https://github.com/contao/contao/blob/f1b3c56856666c38fe080fd04e00797b8fbed310/installation-bundle/src/InstallTool.php#L297

Author: Would you specify that here as well?

The warning in the install tool also only shows this option:

![mysql-strict-mode](https://user-images.githubusercontent.com/1218766/130322519-2f147e1d-9131-41fe-9a28-ed167a49685c.png)

Reviewer 3: @xchs the key is automatically determined by the install tool. So depending on your connection, it will either show `1002` or `3`.

Author: Thank you, all right. Then it will be a bit more cumbersome to document this accordingly... :)

                    engine: MyISAM
```

## MySQL Strict Mode

It is recommended to run MySQL in "strict mode" to prevent corrupt or truncated
data and to guarantee data integrity. As of Contao 4.12, the install tool shows
a warning if the database server is not running in strict mode. To enable it,
add the following to your `my.cnf` file or make sure that the setting is adjusted
accordingly:

```
[mysqld]
…

> sql_mode="STRICT_TRANS_TABLES"

Reviewer 2: ```suggestion
sql_mode="TRADITIONAL"
```

We should probably suggest the `TRADITIONAL` mode here too as we do in the connection settings.

Author: > We should probably suggest the `TRADITIONAL` mode here too as we do in the connection settings.

But the hint in the install tool says "Bitte aktivieren Sie den **Strict-Mode** entweder in Ihrer `my.cnf`-Datei …", so I thought it would be recommended to configure the operation mode to the strict one in the first place. Of course, this assumes that you actually have access to the `my.cnf` or `my.ini` MySQL configuration file.

Reviewer 3: `TRADITIONAL` already includes the following:

```
STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
```

`TRADITIONAL` is the default and recommended setting.

Author: Thank you, then of course it makes sense. 👍🏻

…
```

If the setting cannot be enabled on your server, please configure the connection
options in your `config/config.yml` file:

```yml
doctrine:
    dbal:
        connections:
            default:
                options:

>                     1002: "SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))"
> ```
> 
> Note: For MySQLi the key is named `3` instead of `1002` which is shown only if PDO is used.
> 

Reviewer: ```suggestion
                    # Depending on the DB driver, the option key is either 1002 (pdo_mysql) or 3 (mysqli)
                    1002: "SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))"
```
```
